litmus-portal/frontend/src/pages/MonitoringDashboardPage/index.tsx

```diff
@@ -321,15 +307,23 @@ const DashboardPage: React.FC = () => {
                   history.push('/analytics/dashboard/configure');
                 }}
               >
-                <div>Re-configure dashboard</div>
+                <div>
+                  {t(
+                    'analyticsDashboard.monitoringDashboardPage.reConfigureDashboard'
+                  )}
+                </div>
               </ButtonFilled>
               <ButtonFilled
                 variant="success"
                 onClick={() => {
                   history.push('/analytics/datasource/configure');
                 }}
               >
-                <div>Update data source</div>
+                <div>
+                  {t(
+                    'analyticsDashboard.monitoringDashboardPage.updateDataSource'
+                  )}
+                </div>
```

Comment:
Same Typography/ Litmus-ui text

Reply:
There is no styling required here, is there any use case for using the components?